pkg/cmd/server/start/start_node.go

```diff
@@ -220,57 +221,39 @@ func (o NodeOptions) CreateCerts() error {
 		SerialFile: admin.DefaultSerialFilename(o.NodeArgs.CertArgs.CertDir, "ca"),
 	}
 
-	serverCertInfo := admin.DefaultNodeServingCertInfo(o.NodeArgs.CertArgs.CertDir, o.NodeArgs.NodeName)
-	nodeServerCertOptions := admin.CreateServerCertOptions{
-		GetSignerCertOptions: getSignerOptions,
-
-		CertFile: serverCertInfo.CertFile,
-		KeyFile:  serverCertInfo.KeyFile,
-
-		Hostnames: []string{o.NodeArgs.NodeName},
+	var dnsIP string
+	if len(o.NodeArgs.ClusterDNS) > 0 {
+		dnsIP = o.NodeArgs.ClusterDNS.String()
 	}
 
-	if err := nodeServerCertOptions.Validate(nil); err != nil {
-		return err
-	}
-	if _, err := nodeServerCertOptions.CreateServerCert(); err != nil {
+	masterAddr, err := o.NodeArgs.KubeConnectionArgs.GetKubernetesAddress(&o.NodeArgs.DefaultKubernetesURL)
+	if err != nil {
 		return err
 	}
 
-	clientCertInfo := admin.DefaultNodeClientCertInfo(o.NodeArgs.CertArgs.CertDir, o.NodeArgs.NodeName)
-	mintNodeClientCert := admin.CreateNodeClientCertOptions{
+	nodeConfigDir := path.Join(o.NodeArgs.CertArgs.CertDir, admin.DefaultNodeDir(o.NodeArgs.NodeName))
```

Comment:
hmmm... this stomps any config under openshift.local.certificates/node-<name>, regardless of whether --write-config was specified. Feels like that interacts weirdly if they do `openshift start node --config=openshift.local.config/node-<name>/node-config.json` pointing at a customized config